### ISSUE TITLE
Missing positive glob

### ISSUE THREAD

**Issue by TCB13** [cgross/gulp-dom-src#5]
I'm getting the following error:

`[12:57:02] Error: Missing positive glob`

Guess there was some BC on gulp or a dependency causing this? Thank you.

**Comment by trisys3** [comment]
This usually happens when no files are given to the stream. It's a stupid rule in my opinion, but there you go.

**Comment by linusok** [comment]
@trisys3 

yes, i also find it.  this error comes when we support a tag selector not exist in target html file.

for example:
gulp.task('test',function(){
    console.log('test');
    return gulp.src(‘./app/xx.html’,function(err,files){
        files.map(function(entry){
            console.log(entry);
            var fileName = path.basename(entry,".html");
            var cwd=process.cwd();
            var relativePath = path.relative(cwd,entry);
            var fullPath=path.join(cwd,relativePath);
            if(!relativePath) return;
            //if target html file does not have the selector 'img'，it will come a error 'missing positive glob'.it is bad exactly when processing the batch html files. it is also a faulty that not support the physical path 
            var ret = domSrc({file:relativePath,selector:'img',attribute:'src'});
            ret.pipe(gulp.dest('dist/images'));            
        })
    })
});
